data/mods/Aftershock/items/armor/winter_suits.json

```diff
@@ -0,0 +1,126 @@
+[
+  {
+    "id": "afs_magellan_suit",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "Magellan exosuit" },
+    "description": "A high-quality, civilian grade EVA suit often employed by well-established frontier research and exploration associations.  Designed to support the exploration of challenging terrain, it offers respectable protection against common environmental hazards like extreme temperatures, inhospitable atmospheres, and light radiation.  It leaves arms and hands relatively unencumbered to aid the manipulation of scientific instruments.\n\nAn integral battery allows the suit to operate for up to 34 hours, but complicates field recharging.",
+    "weight": "7800 g",
+    "volume": "14 L",
+    "price": "4 kUSD",
+    "material": [ "nomex", "steel" ],
+    "symbol": "[",
+    "looks_like": "robofac_enviro_suit",
+    "color": "light_gray",
+    "ammo": "battery",
+    "charges_per_use": 1,
+    "use_action": {
+      "type": "transform",
+      "msg": "You activate your %s.",
+      "target": "afs_magellan_suit_on",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "The %s's batteries are dead."
+    },
+    "armor_portion_data": [
+      { "covers": [ "torso" ], "coverage": 100, "encumbrance": 25 },
+      { "covers": [ "leg_l", "leg_r" ], "coverage": 100, "encumbrance": 25 },
+      { "covers": [ "arm_l", "arm_r" ], "coverage": 100, "encumbrance": 15 },
+      { "covers": [ "hand_l", "hand_r" ], "coverage": 100, "encumbrance": 10 },
+      { "covers": [ "foot_l", "foot_r" ], "coverage": 100, "encumbrance": 15 }
+    ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 1000 } } ],
+    "warmth": 20,
+    "material_thickness": 2,
+    "valid_mods": [ "steel_padded" ],
+    "environmental_protection": 15,
+    "flags": [
+      "VARSIZE",
+      "WATERPROOF",
+      "GAS_PROOF",
```

Comment:
Same question above about gas proof.

Reply:
This one could probably keep the passive protection while off, since it doesnt cover the face-head and has no need to let air through